subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testng/TestNGTestFrameworkIntegrationTest.groovy

```diff
@@ -38,8 +42,20 @@ class TestNGTestFrameworkIntegrationTest extends AbstractTestFrameworkIntegratio
                 public void ${passingTestCaseName}() {}
             }
         """
+    }
 
-        TestNGCoverage.enableTestNG(buildFile)
+    @Override
+    void createEmptyProject() {
+        file("src/test/java/NotATest.java") << """
+            public class NotATest {}
+        """
+    }
+
+    @Override
+    void changeTests() {
```

Comment:
I think this should really be called "renameTest" to explain better what type of change it's performing.